lib/rmt/cli/products.rb

```diff
@@ -60,11 +60,11 @@ def enable(*targets)
 
 Examples:
 
-`rmt-cli products disable SLES/15`
+$ rmt-cli products disable SLES/15
```

Comment:
`$ ` is definitly better than the backticks, but I think it would be even cleaner it you just used 2 spaces as indentation like:

```
Examples:
  rmt-cli products disable SLES/15
  rmt-cli products disable 1575
  rmt-cli products disable SLES/15/x86_64 1743
```

Not sure about having newlines in between either. To me, it looks nicer without. But your choice I guess :)